packages/plugin-commands-import/src/import.ts

```diff
@@ -96,6 +132,21 @@ function getAllVersionsByPackageNames (
   }
 }
 
+function getAllVersionsByYarnLockFile (
```

Comment:
Probably it should be 
```suggestion
function getAllVersionsFromYarnLockFile (
```